client/me/purchases/wordads-eligibility-warning-dialog/index.jsx

```diff
@@ -0,0 +1,64 @@
+import { Dialog } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import FormSectionHeading from 'calypso/components/forms/form-section-heading';
+
+import './style.scss';
+
+const WordAdsEligibilityWarningDialog = ( {
+	closeDialog,
+	isDialogVisible,
+	planName,
+	removePlan,
+} ) => {
+	const translate = useTranslate();
+
+	const buttons = [
+		{
+			action: 'cancel',
+			label: translate( 'Cancel' ),
+		},
+		{
+			action: 'remove',
+			isPrimary: true,
+			label: translate( 'Remove Plan' ),
+			onClick: removePlan,
+		},
+	];
+
+	return (
+		<Dialog
+			buttons={ buttons }
+			className="wordads-eligibility-warning-dialog"
+			isVisible={ isDialogVisible }
+			onClose={ closeDialog }
+		>
+			<>
+				<FormSectionHeading>
+					{ translate( 'Remove %(plan)s', {
+						args: { plan: planName },
+					} ) }
+				</FormSectionHeading>
+				<p>
+					{ translate(
+						'When you downgrade your plan, you will become ineligible for the WordAds program. Visit {{a}}our FAQ{{/a}} to learn more.{{br/}}{{br/}}',
```

Comment:
Same here: the BR tags should not be part of the string sent for translation. Just put them outside the string.